Library/OcBootManagementLib/ImageLoader.c

```diff
@@ -835,6 +836,7 @@ InternalEfiLoadImage (
  #else
   mImageLoaderCaps = OC_KERN_CAPABILITY_K64_U64;
  #endif
+  mImageLoaderCapsHandle = NULL;
```

Comment:
I would set this way earlier. While it would normally make sense to only set when we actually load, setting it earlier would allow to catch bugged or weird Load/Start mismatches that would otherwise be masked.